@vates/nbd-client/.USAGE.md

```diff
@@ -0,0 +1,19 @@
+### `new NdbClient({address, exportname, secure = true, port  = 10809})`
+
+create a new nbd  client
+
+
+```js
+
+  import NbdClient from '@vates/nbd-client'
+  const client = new NbdClient({
+    address: 'MY_NBD_HOST',
+    exportname: 'MY_SECRET_EXPORT',
+    cert: 'Server certificate', // optional, will use encrypted link if provided
+  })
+
+  await client.connect()
+  const block = await client.readBlock(blockIndex, BlockSize)
+  await client.disconnect()
+
+```
```

Comment:
Call `./scripts/normalize-packages.js` to generate the `README.md` from this file.